wireguard-install.sh

```diff
@@ -17,21 +25,25 @@ function get_free_udp_port
     fi
 }
 
+# Check EUID
 if [[ "$EUID" -ne 0 ]]; then
     echo "Sorry, you need to run this as root"
     exit
 fi
 
+# Check TUN
 if [[ ! -e /dev/net/tun ]]; then
     echo "The TUN device is not available. You need to enable TUN before running this script"
     exit
 fi
 
-
-if [ -e /etc/centos-release ]; then
+# Check distro and version
+if [ $id == CentOS ]; then
     DISTRO="CentOS"
-elif [ -e /etc/debian_version ]; then
-    DISTRO=$( lsb_release -a 2>/dev/null| grep "Distributor ID" | awk '{print $3}' )
+elif [ $id == debian ]; then
+    DISTR0="Debian"
```

Comment:
this is 0 the number and not O the letter!

Reply:
With much shame and embarrassment I have fixed the typo.